cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -465,6 +498,9 @@ objects:
             subPath: cloud-region-constraints.yaml
           - name: azure-credentials
             mountPath: /secrets/azure-credentials
+          - name: keyvault
+            mountPath: "/secrets/keyvault"
+            readOnly: true
           env:
```

Comment:
Can we also add the flag `azure-first-party-application-certificate-bundle-path` since the file path is now known? 🙏  You may need to update the image tag of cs to an image tag that has this change, similar to what's done in this mr: https://github.com/Azure/ARO-HCP/pull/510/files

The latest tag in that repo `app-sre/uhc-clusters-service` is currently `1c6873c`, which matches the latest commit on cs master. Any reason not to use the latest tag @miguelsorianod @bennerv @janboll ?

Reply:
It looks added here.  I don't see a reason not to use it if CS has functionality for it.